### ISSUE TITLE
Popis situace č.36

### ISSUE THREAD

**Issue by raugustyn** [raugustyn/TB04CUZK001_KnowledgeDiscussions#72]
nezobrazení podměrečného stromořadí (Z_Vegetace_L)
![](http://www.vugtk.cz/euradin/TB04CUZK001/03_CartographicSituations/web/ThumbnailImages/Situace_32.png)

[Plné rozlišení](http://www.vugtk.cz/euradin/TB04CUZK001/03_CartographicSituations/ZM10/Situace_32.png) | [Karta situace](http://www.vugtk.cz/euradin/TB04CUZK001/03_CartographicSituations/web/Situation_36.html) | [Aktualizuj kartu situace](http://www.vugtk.cz/euradin/TB04CUZK001/03_CartographicSituations/web/TB04CUZK001_KnowledgeBase/rest.py/Build/SituationCard/36)

Poslední komentář před uzavřením Issue, uvozený textem 'Popis situace:',  bude použit jako popis situace!

**Comment by droyda** [comment]
Vypuštění prvku pod hranicí treshhold.


Poznámka: já bych zrovna v tomto případě tam ten stromeček dal - všude v okolí je dostatek místa.

**Comment by raugustyn** [comment]
@premekjindrak: Jen abych si byl jistý, mluvíme o stromořadí viz šipka, ze kterého je vidět jen jeden strom???

![687474703a2f2f7777772e767567746b2e637a2f6575726164696e2f5442303443555a4b3030312f30335f436172746f67726170686963536974756174696f6e732f7765622f5468756d626e61696c496d616765732f536974756163655f33322e706e67](https://cloud.githubusercontent.com/assets/3672235/10426651/69745506-70e2-11e5-854c-d78da38c782f.png)

**Comment by droyda** [comment]
Ano, to stromořadí podle dat pokračuje i na druhé straně polní cesty, je tam dost místa pro uvedení značky a může to být i orientační prvek.

**Comment by premekjindrak** [comment]
Opravdu máme na mysli prvek označený šipkou. V katalogu mapových značek je popis specifikující, jak nakládat s touto značkou:
V případě kolize stromořadí s jiným prvkem, např. terénním stupněm, se upraví poloha stromořadí tak, aby byl zachován vztah obou prvků podle vstupních dat ZABAGED. Má-li stromořadí ležet v
terénním stupni, např. na hrázích rybníků, neodsouvá se z příčných čárek, ty jsou automaticky v
mapě odmaskovány. Vyskytuje-li se omylem stromořadí, úzký pruh lesa v lesní půdě se stromy nebo křovinatým porostem, kosodřevinou nebo v zahradě či parku, zneviditelníme ho. Při kolizi stromořadí a bodových značek nebo objektů na komunikacích se stromořadí posune. Podměrečné stromořadí, které má pouze jedno kolečko, se zneviditelní. Při souběhu komunikací, vod, terénních stupňů a liniové vegetace, který nelze vyřešit odsunem, je stromořadí první prvek, který se vypustí.

**Comment by raugustyn** [comment]
hm...jako že obrázek použitý v situaci je chybný, protože ani to jedno kolečko tam ve skutečnosti nemá být??? Tam nějak nevidím funcki, kterou bychom měli vyvíjet. Jednoduše se zneviditelní příliš krátké stromořadí.

Možná to chápe @janlangr???

**Comment by zemekjar** [comment]
Dva liniové objekty stromořadí jsou v modelu samostatnými objekty. Je však potřebné na ně nahlížet, jako na sebe navazující části téhož objektu. Tato operace představuje vytváření vyšší struktury, což je opakem operace segmentace. Vyššími strukturami máme na mysli: shluky bodových nebo areálových objektů, říční sítě, silniční tahy nebo sítě, uskupení budov c blocíchů, vytváření hranic intravilánu apod.

**Comment by raugustyn** [comment]
Které dva objekty? Jarku stáhni si pomocí pravého tlačítka změnšený náhled úplně nahoře v tomhle issue, otevři v kreslení, namaluj šipku nebo něco, vstup do editace tvého postu (tužtička) a potom myší tam strč ten obrázek. Díky.

**Comment by janlangr** [comment]
Pokud platí výše uvedené pravidlo, pak situace zobrazuje jeho chybnou interpretaci. Definovaný liniový symbol má takovou vzdálenost bodů na linii, že v tomto případě ani při agregaci do vyšší strukrury zřejmě nebude symbol schopen vyjádřit tvar stromořadí. Navíc body na linii stromořadí nemohou kolidovat se symbolem polní cesty, kterou by agregovaný prvek protínal. Proto bych se k agregaci neklonil. Určitě je nezbytné stanovit prahovou délku linie nutnou pro vykreslení.
